### DISCUSSION TITLE
如何获取定时参数 （How to Obtain Timing Parameters）

### DISCUSSION THREAD

**Discussion by Chenslan** [apache/doris#42679] [A - General / Q&A] [answered]
Starting from version 2.1, Doris can create scheduled tasks at a specific time. How can I get the scheduled time? The official example uses the `now()` function, but I wonder if the `now()` function might return a different scheduled time due to network latency or other reasons?

![1](https://github.com/user-attachments/assets/9ad427c7-473a-46ae-984a-5d835081c963)
![2](https://github.com/user-attachments/assets/ec28c98c-b7d8-4f09-a328-dbbe0351e03d)

**Comment by ixzc** [comment, marked as answer]
yes, it will be return a different scheduled time due to network latency, if you want to use a precise time, you need to use a variable, which will be support in the future.
